openstudiocore/src/utilities/core/UUID.i

```diff
@@ -12,16 +12,13 @@
 namespace openstudio{
 
   #ifdef SWIGRUBY
-    %rename("nil?") isNull();
+    %rename("nil?") is_nil();
```

Comment:
I cannot imagine why it wouldn't work? It's been that way for eons AFAICT. All I did was change the name to match the new uuid class.